pgmpy/estimators/MLE.py

```diff
@@ -68,10 +68,8 @@ def get_parameters(self):
             else:
                 parent_card = np.array([self.node_card[parent] for parent in parents])
                 var_card = self.node_card[node]
-                state_counts = self.data.groupby([node] + self.model.predecessors(node)).count()
-                values = state_counts.iloc[:, 0].reshape(var_card,
-                                                         np.product(parent_card))
-                cpd = TabularCPD(node, var_card, values,
+                values = self.data.groupby([node] + parents).size().unstack(parents).fillna(0)
```

Comment:
@yashu-seth I can't think of a case when we will get a `NaN`. Do you have any case in mind ? Otherwise we won't be needing the `fillna` call.